docs/dev/ci-reference.adoc

```diff
@@ -36,11 +36,23 @@ If you need to run `odo` integration tests against a specific version of Openshi
 
 == Running integration tests on Prow
 
-Prow is the Kubernetes or OpenShift way of managing workflow, including tests. To get tests, you need to raise PR to `openshift/release` repository setting up appropriate ci operator config and job files. Reference for the same is available in the repository.
+Prow is the Kubernetes or OpenShift way of managing workflow, including tests. Odo integration and periodic test targets are passed through the script `scripts/openshiftci-presubmit-all-tests.sh` and scripts/openshiftci-periodic-tests.sh respectively available in the odo https://github.com/openshift/odo/tree/master/scripts[repository]. Prow uses the script through the command attribute in the odo job configuration file in https://github.com/openshift/release/tree/master/ci-operator/config/openshift/odo[openshift/release] repository. Reference for the same is available in the repository.
 
-Note that prow gives you a bare bones cluster, so we need to pre-configure the same so that the cluster is in state expected such as auth being configured and so on.
+For running integration test on 4.5 cluster, job configuration file will be
+[source,sh]
+----
+    tests:
+    - as: integration-e2e
+      commands: scripts/openshiftci-presubmit-all-tests.sh
```

Comment:
This is for `openshiftci-presubmit-all-tests.sh`. Can we also add the source for `scripts/openshiftci-periodic-tests.sh`. I mean how to configure periodic jobs

Reply:
done